inspirehep/modules/workflows/search.py

```diff
@@ -39,8 +39,10 @@ def holdingpen_search_factory(self, search, **kwargs):
     search, urlkwargs = default_search_factory(self, search, **kwargs)
     includes = [
         'metadata.titles', 'metadata.abstracts', 'metadata.authors',
-        'metadata.inspire_categories', 'metadata.name', 'metadata.positions',
-        'metadata.acquisition_source', 'metadata.arxiv_eprints',
+        'metadata.earliest_date', 'metadata.publication_info',
```

Comment:
as I commented earlier, these changes are not relevant to matches? can you make sure if they are needed. Perhaps this is related to https://github.com/inspirehep/inspirehep-search-js/pull/78/files#r181077460

Reply:
I tried excluding them. When I do that, these fields don't show up on the UI (as shown below)
![localhost_5000_holdingpen_list__page 1 size 10 status halted 1](https://user-images.githubusercontent.com/11242410/38721938-74df93b6-3efc-11e8-9be1-f0463adbbca1.png)

Reply:
So, I checked again. These fields are needed to display information for the top record (the record for which we are displaying the matches).

Reply:
yeah ok I see now, changes in the other PR.